docs/usermanual/windows-installation.rst

```diff
@@ -37,11 +39,11 @@ The rocalution.hpp header file must be included in the user code to make calls
 into rocALUTION, and the rocALUTION import library and dynamic link library will become respective link-time and run-time
 dependencies for the user application.
 
-Once installed, find rocalution.hpp in the HIP SDK `\\include\\rocalution`
+Once installed, find rocalution.hpp in the HIP SDK ``\\include\\rocalution``
 directory. Only use these two installed files when needed in user code.
 
 ----------------------------------
-Building and Installing rocALUTION
+Building and installing rocALUTION
 ----------------------------------
 
 Building from source is not necessary, as rocALUTION can be used after installing the pre-built packages as described above.
```

Comment:
"The rocALUTION source code, which is the same as for the ROCm linux distributions, is available at the `rocALUTION github page <https://github.com/ROCmSoftwarePlatform/rocSPARSE>`_." Is this link correct?

Reply:
No, should be https://github.com/ROCmSoftwarePlatform/rocALUTION instead

Reply:
Done